apps/mysql.go

```diff
@@ -299,12 +342,20 @@ func (p LoggingProcessorMysqlGeneral) Components(ctx context.Context, tag string
 			{
 				StateName: "start_state",
 				NextState: "cont",
-				Regex:     `\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d+Z`,
+				Regex: fmt.Sprintf(
+					`^(%s|%s|\t\t)`,
```

Comment:
Does `timeRegexMariaDBNew` need to be included in here too? Or are those logs not multiline?

Reply:
Nevermind, looks like `timeRegexMariaDBNew` only applies to the error log.